core-web/libs/dotcms-scss/angular/dotcms-theme/components/buttons/_button.scss

```diff
@@ -47,6 +47,10 @@
     &.p-button-outlined {
         @extend #button-disabled-outlined;
     }
+
+    &.p-button-text {
+        @extend #button-disabled-text;
+    }
```

Comment:
Screenshot to see what's this is doing.

Reply:
This makes from:
this
<img width="91" alt="Screenshot 2023-11-21 at 9 49 47 AM" src="https://github.com/dotCMS/core/assets/1909643/10bc6f0e-fcd5-42d6-9e5b-3c707f49665e">

to this:
<img width="91" alt="Screenshot 2023-11-21 at 9 49 55 AM" src="https://github.com/dotCMS/core/assets/1909643/e15e4f76-303e-480e-b4e8-aabdbe75724c">